rosie/src/main/java/com/karumi/rosie/domain/usecase/UseCaseHandler.java

```diff
@@ -45,8 +53,12 @@ void execute(RosieUseCase useCase, UseCaseParams useCaseParams) {
     UseCaseFilter.filter(useCase, useCaseParams);
 
     useCase.setOnSuccessCallback(useCaseParams.getOnSuccessCallback());
-    useCase.setOnErrorCallback(
-        new OnErrorCallbackToErrorHandlerAdapter(errorHandler, useCaseParams.getOnErrorCallback()));
+
+    OnErrorCallbackToErrorHandlerAdapter errorAdapter =
```

Comment:
What variable name do you prefer?
OnErrorCallback "errorAdapter" or "onErrorCallback" ?
private final List<OnErrorCallback> "errorAdapters" or "errorCallbacks" ?